### DISCUSSION TITLE
Users can login admin dashboard?

### DISCUSSION THREAD

**Discussion by onurdev74** [directus/directus#2751] [Archive]
User can sign up to `directus_users` table also can login dashboard. How to prevent login to dashboard? I created custom users table but `POST /:project/auth/authenticate` and other request using  `directus_users` table. Any advice? I think should be wrap tables by developer like `admin_users` `normal_users`

**Comment by rijkvanzanten** [comment]
There's no way to prevent Directus users from logging in to the app. We're working on adding this as a feature in the next major version though

**Reply by vatologic** [reply]
Even though the API would give the same level of access as the dashboard, it just doesn't feel right to have users logging into the panel. Kind of defeats the point of having the front-end decoupled from the back-end. So, some kind of checkbox at the role level to allow/deny access would be greatly appreciated.

**Reply by benhaynes** [reply]
Agree — and it has been added to v9. 👍

**Reply by vatologic** [reply]
I kind of got that from @rijkvanzanten - but to get reaffirmation makes Directus just a tiny bit more awesome. Really looking forward to the stable release of v9. Thanks!
